packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -1288,24 +1290,28 @@ class ListView extends BoxScrollView {
 /// Here are two brief snippets showing a [GridView] and its equivalent using
 /// [CustomScrollView]:
 ///
+/// ![A screenshot of a GridView](https://flutter.github.io/assets-for-api-docs/assets/widgets/grid_view.png)
+///
 /// ```dart
 /// GridView.count(
 ///   primary: false,
 ///   padding: const EdgeInsets.all(20.0),
 ///   crossAxisSpacing: 10.0,
+///   mainAxisSpacing: 10.0,
 ///   crossAxisCount: 2,
 ///   children: <Widget>[
-///     const Text('He\'d have you all unravel at the'),
-///     const Text('Heed not the rabble'),
-///     const Text('Sound of screams but the'),
-///     const Text('Who scream'),
-///     const Text('Revolution is coming...'),
-///     const Text('Revolution, they...'),
+///     Container(child: const Padding(padding: EdgeInsets.all(8.0), child: Text('He\'d have you all unravel at the')), color: Colors.teal[100]),
```

Comment:
```suggestion
///     Container(
///       child: const Padding(
///         padding: EdgeInsets.all(8.0), 
///         child: Text('He\'d have you all unravel at the'),
///        ), 
///        color: Colors.teal[100],
///      ),
```
Here and below for style guide compliance / readability. :)